independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/InterceptorInfo.java

```diff
@@ -102,32 +132,32 @@ public Set<AnnotationInstance> getBindings() {
         return bindings;
     }
 
-    public MethodInfo getAroundInvoke() {
-        return aroundInvoke;
+    public List<MethodInfo> getAroundInvokes() {
```

Comment:
Note that `InterceptorInfo` is part of the API. I think that we should deprecate the singular forms of  methods and return the method at the top in the hierarchy, i.e. the last method from the list.